ada-project-docs/deploy.md

```diff
@@ -0,0 +1,59 @@
+# Deploying to GitHub Pages with `parcel-bundler`
+
+Follow the steps below to deploy to GitHub Pages using `parcel-bundler`
+
+1. Create a new branch called `deployed-solution`
```

Comment:
Is this just so that the changes don't affect their main submission code (and ability to run locally)?
```suggestion
1. Create a new branch called `deployed-solution` and switch to it.
```